nginx.conf

```diff
@@ -0,0 +1,14 @@
+user  nginx;
+
+events {
+    worker_connections   1000;
+}
+
+http {
+    server {
+        listen 8080;
+        location / {
+            proxy_pass http://wombat:8080;
```

Comment:
I don't understand where the load balancing logic resides. The upstream is a black box from nginx's perspective, is that right? I also didn't see that `docker-compose up --scale` implies load balancing.

Reply:
Ah, good point! I should add an explanatory note. Docker compose uses round robin dns, so `wombat` should resolve to each of the wombat instances.